Cargo.toml

```diff
@@ -235,8 +235,20 @@ webbrowser = "0.8"
 winit = { version = "0.29.9", default-features = false }
 # TODO(andreas): Try to get rid of `fragile-send-sync-non-atomic-wasm`. This requires re_renderer being aware of single-thread restriction on resources.
 # See also https://gpuweb.github.io/gpuweb/explainer/#multithreading-transfer (unsolved part of the Spec as of writing!)
-wgpu = { version = "0.18.0", features = ["fragile-send-sync-non-atomic-wasm"] }
-wgpu-core = "0.18.0"
+wgpu = { version = "0.19.1", default-features = false, features = [
+  # Backends:
+  "webgl",
+  "metal",
+  "webgpu",
+  "webgl",
```

Comment:
What about "dx12" and "angle" ?

https://github.com/gfx-rs/wgpu/blob/trunk/wgpu/Cargo.toml#L29-L53